CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# 0.35.0
+
+- Propose-downstream: log when a PR already exists downstream (#1322).
+- `packit init` to set spec file path in the config if it's not defined (#1313).
+- Don't use `str.removeprefix()` since it was introduced in Python 3.9 and we still support older versions of Python 3 (#1318).
+- Make it possible to clone packages from staging dist-git (#1306).
+- Source-git: squash patches by patch name - no need to have a dedicated attribute, `squash_commits`, for that (#1309).
+- Source-git: look for the config file in .distro/source-git.yaml as well (#1302).
+- Source-git: don't raise an exception when a patch with final name already exists (#1316).
+- Source-git: change logging from error to warning when %prep is not using %(auto)setup (#1317).
+- Add a Pull Request template to the project to ease the contribution process (#1323).
```

Comment:
not user facing :)

Reply:
so where do we put this kind of info - blog post? I totally want to have it somewhere; agreed those it's useless in the rpm %changelog

Reply:
from my POV it's technical detail related to our workflow
ad blog post, sure if we targeted our contributors with it; and from the JIRA task or docs in kwaciaren I deduce we're targeting user base :)